build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -203,7 +203,6 @@ spec:
               path: /readyz
               port: api
               scheme: HTTPS
-            initialDelaySeconds: 5
```

Comment:
The doc https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes seems wrong about "If the value of periodSeconds is greater than initialDelaySeconds then the initialDelaySeconds will be ignored. Defaults to 0 seconds. Minimum value is 0."

From the implementation of [prober](https://github.com/kubernetes/kubernetes/blob/66e34012255abf1bbd0956a712817dad77c69c41/pkg/kubelet/prober/worker.go#L146), `initialDelaySeconds` is used to skip a probe and `periodSeconds` is used to determine interval between probes, and the first probe attemp doesn't wait for `periodSeconds`. If `initialDelaySeconds` is removed, it would probe immediately after prober starts, which is not wanted. 

Previously it could tolerate at least 5s+70s of disconnection. After removing the initialDelaySeconds, it would tolerate 70s.

The effect of `initialDelaySeconds` can be confirmed by checking the timestamps of the following Pod:
```
apiVersion: v1
kind: Pod
metadata:
  name: nginx-pod
  labels:
    app: nginx
spec:
  containers:
  - name: nginx
    image: nginx:latest
    ports:
    - containerPort: 80
    readinessProbe:
      exec:
        command:
        - /bin/sh
        - -c
        - "date >> /tmp/probes"
      initialDelaySeconds: 15 
      periodSeconds: 20     
      successThreshold: 1  
      failureThreshold: 3
```
The container start time was "2024-10-23T14:54:10Z", the 1st probe was "Wed Oct 23 14:54:27 UTC 2024".

If `initialDelaySeconds` is removed, the container start time was "2024-10-23T14:53:03Z", the 1st probe was "Wed Oct 23 14:53:05 UTC 2024".